Subaharish/HTML BASICS/assignment3.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Registration</title>
+    </head>
+    <body>
+        <h1>Register</h1>
+        <form>
+            First name:
+            <input type="text" name="Firstname"  placeholder="Firstname" required>
+            Last name:
+            <input type="text" name="Lastname" placeholder="Lastname" required>
+            <br>
+            <br>
+            Gender:
+            <label>Male:<input type="radio" name="gender" value="male" checked></label>
+            <label>Female:<input type="radio" name="gender" value="female"></label>
+            <label>Other:<input type="radio" name="gender" value="other" ></label>
```

Comment:
Refer this: [HTML <label> Tag - w3schools.com](https://www.w3schools.com/tags/tag_label.asp)